README.md

```diff
@@ -39,6 +39,11 @@ You can use python standard [logging](https://docs.python.org/3/library/logging.
 >>> logger.addHandler(ch)
 ```
 
+
+### ReadTheDocs
+
+You can find the detailed documantation of the package on the official website of [ReadTheDocs](https://vcf2fhir.readthedocs.io/en/latest/).
```

Comment:
The official documentation is hosted on ReadTheDocs.io: [https://vcf2fhir.readthedocs.io/en/latest/](https://vcf2fhir.readthedocs.io/en/latest/)